src/GoTrueApi.ts

```diff
@@ -347,6 +348,43 @@ export default class GoTrueApi {
     }
   }
 
+  /**
+   * Send User supplied Email / Mobile OTP to be verified
+   * @param email The user's email address
+   * @param phone The user's phone number WITH international prefix
+   * @param token token that user was sent to their mobile phone
+   * @param type verification type that the otp is generated for
+   * @param redirectTo A URL or mobile address to send the user to after they are confirmed.
+   */
+  async verifyEmailOrMobileOTP(
```

Comment:
Can we just call this `verifyOTP` and then mark `this.api.verifyMobileOTP` as deprecated?

Reply:
do we still want to keep [verifyOTP](https://github.com/supabase/gotrue-js/pull/251/files#diff-3522461172efd6058d6b8da62fc2d30d8b524d2b64894ea2c67218c52f7fdff5R267) in `GoTrueClient`?

Reply:
I'm with marking `this.api.verifyMobileOTP` as deprecated but it's being used in `verifyOTP` in `GoTrueClient`. How should we differentiate the `verifyOTP` in `GoTrueClient` and `GoTrueApi`?